File: app/src/main/java/com/digiapps/installedapps/MainActivity.java
package com.digiapps.installedapps;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.provider.Settings;

public class MainActivity extends Activity {
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        finishAndRemoveTask();
        startActivity(new Intent(Settings.ACTION_APPLICATION_SETTINGS));
    }
}

File: app/src/main/java/com/digiapps/installedapps/MainActivity.kt
package com.digiapps.installedapps

import android.content.ComponentName
import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val intent = Intent(Intent.ACTION_MAIN)
        intent.component = ComponentName(
            "com.android.settings",
            "com.android.settings.applications.ManageApplications"
        )
        startActivity(intent)
        finish()
    }
}
